packages/cf-component-text/src/TextTheme.js

```diff
@@ -1,7 +1,8 @@
 export default baseTheme => ({
   fontSizeNormal: '1em',
-  fontWeightNormal: 400,
   fontSizeSmall: '.8em',
+  fontWeightNormal: 400,
+  fontWeightBold: 600,
```

Comment:
As per our variable.js bold should be 700. Could we also add a semi-bold to have completeness?  
https://github.com/cloudflare/cf-ui/blob/master/packages/cf-style-const/src/variables.js#L25-L28